.github/workflows/ci-lint-test.yml

```diff
@@ -68,5 +69,31 @@ jobs:
           echo "Generating keypair..."
           solana-keygen new -o "$HOME/.config/solana/id.json" --no-passphrase --silent
 
-      - name: Run tests
+      - name: Run bpf tests
         run: cargo test-bpf
+
+        # Create a cache for Soteria
+      - name: Cache Soteria
+        id: cache-soteria
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/soteria
+          key: ${{ runner.os }}-soteria
+          
+        # Install Soteria  
+      - name: Install Soteria
+        run: |
```

Comment:
I think we need an need an `if: steps.cache-soteria.outputs.cache-hit != 'true'` here. It's also missing for the solana-bin installation above.

Reply:
I dropped it to do a fresh install for each run as the install step itself is relatively fast. Happy to add it back in, but risks running an older cached version